charmcraft/utils.py

```diff
@@ -192,13 +192,21 @@ def create_manifest(basedir, started_at):
     # we integrate lifecycle lib in future branches
     architectures = [ARCH_TRANSLATIONS.get(os_platform.machine, os_platform.machine)]
 
+    # XXX Facundo 2021-04-19: these are temporary translations until charmcraft
+    # changes to be a "classic" snap
+    name_translation = {'ubuntu core': 'ubuntu'}
```

Comment:
Shouldn't this be `ubuntu-core` instead since it translates the value in the os release ID?

Reply:
Fixed, thanks.